bin/root-transition.sh

```diff
@@ -151,19 +153,69 @@ transition() {
   echo "Please save them safely and privately."
 }
 
+check_version_namespace() {
+  local OUT
+  OUT=$(kubectl get po -n $1 -o yaml | grep "istio\/proxyv2\:1\.")
+
+  for LINE in ${OUT};
+  do
+    if [[ ${LINE} == *"istio/proxyv2"* ]]; then
+      LINE=${LINE#"gke.gcr.io/istio/proxyv2:"};
+      LINE=${LINE#"docker.io/istio/proxyv2:"};
+      LINE=${LINE#"gcr.io/gke-release/istio/proxyv2:"};
+      LINE=${LINE#"istio/proxyv2:"};
+      VER=${LINE%%"-gke.0"};
+      echo "Istio proxy version: $VER";
+    fi
+  done
+}
+
+check_version() {
+  local NS
+  NS=$(kubectl get ns | grep -v "STATUS" | grep -v "kube-system" | grep -v \
+    "kube-public" | awk '{print $1}')
+
+  for n in ${NS}
+  do
+    echo "Checking namespace: ${n}"
+    check_version_namespace "${n}"
+  done
+}
+
 case $1 in
-  check)
-    check
+  check-root)
+    check_root
+    ;;
+
+  check-version)
+    check_version
     ;;
 
-  transition)
-    transition
+  root-transition)
+    root_transition
     ;;
 
-  verify)
-    verify
+  verify-certs)
+    verify_certs
     ;;
 
   *)
-    echo $"Usage: $0: check | transition | verify "
+    echo $"Usage: check-root | check-version | root-transition | verify-certs
```

Comment:
You can remove the `$` at the beginning.

Reply:
Done.